Doc/c-api/sys.rst

```diff
@@ -98,9 +98,9 @@ Operating System Utilities
 .. c:function:: int PyOS_CheckStack()
 
    Return true when the interpreter runs out of stack space.  This is a reliable
```

Comment:
```rst
.. index:: single: USE_STACKCHECK (C macro)
```

Reply:
Added.

In addition, there were warnings about `PyOS_sighandler_t` and `PyAuditHookFunction`. The former was described explicitly in two places, the latter not at all. I added `c:type` entries for them. Please check my C type definitions. It has been 30+years since I worried about pointers to functions and was never very good at it back then...